l3kernel/CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ this project uses date-based 'snapshot' version identifiers.
 - `\bitset_use:N` and `\clist_use:N`: this clarifies that bitsets and clists
   can be used with `V`-type expansion
 - `\sys_if_engine_opentype:(TF)`
+- Add equivalent interfaces for regex functions:
+  - `\int_(g)set_regex_count:Nnn`, `\int_(g)set_regex_count:NNn`
+  - `\seq_(g)set_regex_extract:Nnn`, `\seq_(g)set_regex_extract:NNn`
+  - `\seq_(g)set_regex_split:Nnn`, `\seq_(g)set_regex_split:NNn`
+  - `\tl_if_regex_match:nn(TF)`, `\tl_if_regex_match:nN(TF)`
+  - `\tl_regex_(g)replace_once:Nnn`, `\tl_regex_(g)replace_once:NNn`
+  - `\tl_regex_(g)replace_all:Nnn`, `\tl_regex_(g)replace_all:NNn`
```

Comment:
About naming: compared to existing `\tl_(g)replace_(once|all):..` functions, would `\tl_(g)replace_regex_(once|all):...` be better?

`\regex_replace...` is different from `\regex_(count|extract|split)...` in that `replace` updates a tl var in-place.

Reply:
The idea was that they match the names in the `regex` module - it has `\regex_replace_once:nnN`, for example.